mmv1/third_party/terraform/resources/resource_storage_transfer_job.go

```diff
@@ -66,6 +67,32 @@ func resourceStorageTransferJob() *schema.Resource {
 				ValidateFunc: validation.StringLenBetween(0, 1024),
 				Description:  `Unique description to identify the Transfer Job.`,
 			},
+			"loggingConfig": {
+				Type:     schema.TypeList,
```

Comment:
Can multiple logging configs be set? Otherwise we usually cap the number of items with
```
				MaxItems: 1,
```